cartoframes/context.py

```diff
@@ -1099,8 +1099,7 @@ def data_boundaries(self, boundary=None, region=None, decode_geom=False,
             boundaries in `region` (or the world if `region` is ``None``)
         """
         # TODO: create a function out of this?
-        if (isinstance(region, collections.Iterable)
-                and not isinstance(region, str)):
+        if (isinstance(region, collections.Iterable) and not isinstance(region, str)):
```

Comment:
Maybe check the `isintance(str)` first, so we don't have to do the negative check? e.g:

```
if isinstance(str)
   ...
elif isinstance(Iterable)
   ...
```